clients/vscode-hlasmplugin/src/hlasmListingServices.ts

```diff
@@ -340,15 +340,15 @@ function asHover(md: vscode.MarkdownString | undefined): vscode.Hover | undefine
     return md ? new vscode.Hover(md) : undefined;
 }
 
-function isolateSymbolSimple(document: vscode.TextDocument, position: vscode.Position) {
+function isolateSymbolSimple(document: vscode.TextDocument, position: vscode.Position, hasPrefix: boolean) {
     if (position.line >= document.lineCount)
         return undefined;
     const line = document.lineAt(position.line).text;
 
     let start = position.character;
     let end = position.character;
 
-    while (start > 0 && ordchar.test(line[start - 1]))
+    while (start > +hasPrefix && ordchar.test(line[start - 1]))
```

Comment:
I'm almost afraid to ask but is the plus sign in `+hasPrefix` some weird thing in TypeSript?

Reply:
should be just false->0, true->1